### ISSUE TITLE
Make API more translatable

### ISSUE THREAD

**Issue by MGaetan89** [SiCKRAGETV/sickrage-issues#1765]
Hi,

I am working on an Android application for SickRage. I'd like to make my application available in various languages, but the SickRage API returns hardcoded value in English.

For example, when calling `?cmd=show`, I have the following two information:

```json
{
    "airs": "Monday 8:00 PM",
    "status": "Continuing",
}
```

Would it be possible to add more information to ease the translation process?
For example, for `airs`, add a `air_date` field that contains an integer representing the day (`0` for `Monday`, `1` for `Tuesday`, ...) and `air_time` that contains the time of the episode.
For `status`,  maybe a `status_id` which contains an integer ( `0` for `Unaired`, `1` for `Continuing`, ...). Unless you can guarantee that the current values in `status` won't change in the future?

This request is true for other API methods as well. This was just an example.

Thanks

**Comment by MGaetan89** [comment]
Might be related to #979

**Comment by fernandog** [comment]
@MGaetan89 you need to make sure it wont break compability with DroidBead and NZB360 apps. I would add another filelds, not change the existing ones

**Comment by MGaetan89** [comment]
Yes, I agree. The problem is documentation. How will people know what each value means?
Once everything is figured out, I can work on PR and test if DroidBeard and NZB360 are still working as expected.

**Comment by fernandog** [comment]
@MGaetan89 possible. the problem is time. If have only a little to spend on SR. If you want to help, feel free

**Comment by MGaetan89** [comment]
I will have a look and try to implement that. I'll start by the end of the month, beginning of July.

**Comment by OmgImAlexis** [comment]
@MGaetan89 since we discussed moving to /v2 for the api, what if we added this to the new one? We could then send a mass email to all of the main devs for things like android apps letting them know there's a new api if they'd like to transition over to it.

**Comment by MGaetan89** [comment]
Yes, that's fine for me.
